### PR TITLE
Bugzilla 1255342 Adding a new section called Searching for Images, Templates, and Othe…

### PR DESCRIPTION
**Pull request by ghost**
…r Inputs

@openshift/team-documentation PTAL.

### CODE REVIEW

File: dev_guide/application_lifecycle/new_app.adoc [hunk 1]
```diff
@@ -527,6 +527,16 @@ $ oc new-app \
     --group=ruby+mysql
 ----
 
+[[searching-images-templates-inputs]]
+
+==== Searching for Images, Templates, and Other Inputs
+To search for images, templates, and other inputs for the `oc new-app` command, add the `--search` and `--list` flags. For example,
+to find all of the images or templates that include PHP:
```
Comment (ahardin-rh) [inline dev_guide/application_lifecycle/new_app.adoc]: There seems to be a weird line break here. Otherwise, looks good!

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:..dev_guide/application_lifecycle/new_app.adoc#searching-images-templates-inputs[Searching for Images, Templates, and Other Inputs]
|New topic on searching for images, templates, and other inputs when creating new applications.
%

**Comment by ahardin-rh** [comment]
@tmorriso-rh Content looks good!  As for the revision history, you need a /  in the xref path in front of dev_guide. Also, note that this is a new section, not a new topic. Topics are their own pages, while sections are the headings within topics. So, the topic should be Creating New Applications, which is a child topic of Application Life Cycle Management. So, you can revise this portion of the rev history to look like:

|xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[Application Life Cycle Management -> Creating New Applications]
|New section on xref:../dev_guide/application_lifecycle/new_app.adoc#searching-images-templates-inputs[Searching for Images, Templates, and Other Inputs] when creating new applications.

This content will eventually appear in https://docs.openshift.com/container-platform/3.6/dev_guide/revhistory_dev_guide.html, so you can look to that page as an example. Notice that the first part is headed Affected Topic and the second part is the description of the change, which can link to the section. Sorry that this is verbose! Hopefully that this is helpful and makes sense! 😝

**Comment by tmorriso-rh** [comment]
Thanks @ahardin-rh. Thinking in terms of topics and child topics is a good reminder.

**Comment by vikram-redhat** [comment]
@tmorriso-rh - this hasn't been picked to any of the stage branches? You must remember to pick commits to stage branches straight away.

**Pull request merged**